verification/cbmc/proofs/aws_ring_buffer_init/Makefile

```diff
@@ -20,6 +19,8 @@ PROOF_SOURCES += $(PROOF_STUB)/error.c
 PROJECT_SOURCES += $(SRCDIR)/source/ring_buffer.c
 PROJECT_SOURCES += $(SRCDIR)/source/common.c
 
-###########
+# We abstract this function because manual inspection demonstrates it is unreachable.
+# Improves coverage metrics.
+REMOVE_FUNCTION_BODY += __CPROVER_file_local_ring_buffer_c_s_ring_buffer_mem_acquire
```

Comment:
We have a warning in the report for unexpected omitted functions, but no other error. Usually, these function are consider because of function pointers.

Reply:
Thanks, sounds like this is something that CBMC folks need to improve at their end.

At the very least, we should have that flag to bump warnings to errors, so that we can catch this.
Making the CFA more precise would be the best solution of course, but that's hard ...